health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -89,18 +198,120 @@ const FormulaConfiguration = ({onSelect,  category,  customProps})=>{
+                                          variation={"primary"}
+                                          label={t("NO")}
+                                          onButtonClick={()=> {
+                                            setFormulasPopUp(false)
+                                          }}
+                                        />,
+                                      ]}
+                                      sortFooterChildren={true}
+                                      onClose={() => {
+                                        setFormulasPopUp(false)
+                                      }}
+                          ></PopUp> }      
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)